### DISCUSSION TITLE
What is the privacy policy of Deep Java Library

### DISCUSSION THREAD

**Discussion by InAnYan** [deepjavalibrary/djl#3370] [Q&A] [answered]
Hi! I'm developing an app using `djl`.

The app will download models from `djl`'s model zoo (via `djl://...` URL).

What is the privacy policy of `djl` or Model Zoo or the server I'll download from, so I could link that to our users?

**Comment by frankfliu** [comment, marked as answer]
Download from DJL model is anonymous, we don't collect your personal information when using model zoo.
What concern do you have regarding model zoo?

**Reply by koppor** [reply]
The issue is that if an application is distributed to third parties. This application will connect to your servers. Using their cooperate proxies and country regulations. Here in the EU, wie have the GDPR. As a distributor of a software, one needs make the privacy policies of the used services transparent. 

I assume you do not store any IP address of downloaders. Otherwise, that would be processing of personal data (if the [case number C-582/14](https://www.lexology.com/library/detail.aspx?g=2f32162f-214c-41fc-8ab8-997ddf9ba418) is still not overruled by another court).

**Comment by frankfliu** [comment]
DJL model Zoo is hosted by Amazon CloudFront. We can only see statistic data from AWS CloudFront console.
